src/contracts/FoxStaking.sol

```diff
@@ -412,6 +453,18 @@ contract FoxStaking is Ownable {
         warmupContract = _address;
     }
 
+    /**
+        @notice sets the contract address for LP staking
+        @param _address address
+     */
+    function setCooldownContract(address _address) external onlyManager {
```

Comment:
One option to simplify our deployment script would be just to have the FoxStaking.sol constructor create the cooldown and warmup contracts.  We can then remove all of these setters and mark the addresses as constants